packages/libs/components/src/plots/BipartiteNetwork.tsx

```diff
@@ -26,6 +33,12 @@ export interface BipartiteNetworkSVGStyles {
   columnPadding?: number; // space between the left of the svg and the left column, also the right of the svg and the right column.
 }
 
+export interface NodeAction {
```

Comment:
I wonder if this would be better in `Network.tsx`? Or we can leave that for when we consolidate the components, too!

Reply:
Yeah, I'm inclined to wait for consolidation. I was going to move things around now, but decided it was better to wait for a clear path forward.